.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @samwelkanda @shemogumbe @baywet @darrelmiller @ddyett @andrueastman @silaskenneth @ndiritu
```

```suggestion
* @microsoft/kiota-write
```

let's start to align all the repositories